index.html

```diff
@@ -231,23 +239,25 @@ <h1>Skills</h1>
         </fieldset>
 
     </form>
+</div>
+   
+</section>
 
-    <div class="socialmedia">
+<footer class="socialmedia-desk">
 
-        <ul>
-            <li><a href="#"><img src="media/twitter.jpg" alt="twitter_image"></a></li>
-            <li><a href="#"><img src="media/Linkedin.jpg" alt="Linkedin_image"></a></li>
-            <li><a href="#"><img src="media/github.jpg" alt="github_image"></a></li>
-            <li><a href="#"><img src="media/medium.jpg" alt="medium_image"></a></li>
-            <li><a href="#"><img src="media/angular.jpg" alt="angular_image"></a></li>
-                      
-        </ul>
+       
+    <ul>
+        <li><a href="#"><img src="media/twitter.jpg" alt="twitter_image"></a></li>
+        <li><a href="#"><img src="media/Linkedin.jpg" alt="Linkedin_image"></a></li>
+        <li><a href="#"><img src="media/github.jpg" alt="github_image"></a></li>
+        <li><a href="#"><img src="media/medium.jpg" alt="medium_image"></a></li>
+        <li><a href="#"><img src="media/angular.jpg" alt="angular_image"></a></li>
+                  
+    </ul>
 
-    </div>
 
-    <h1 class="mail">myemailhere@email.com</h1>
+<h1 class="mail">myemailhere@email.com</h1>
```

Comment:
- [ ] great job with the contact section. please kindly align the text and the form to align with the cards above it like in the original version. nice work
yours
![Screenshot (825)_LI](https://user-images.githubusercontent.com/65298048/220935335-0015b0dc-5fef-43ec-b562-ad9045c1c32d.jpg)
figma
![Screenshot (826)](https://user-images.githubusercontent.com/65298048/220935578-37d64dd7-2fde-4fe0-8b28-28a4b9c93bf3.png)